pkg/pool-utils/test/AumProtocolFeeCache.test.ts

```diff
@@ -56,17 +54,17 @@ describe('AumProtocolFeeCache', () => {
       });
 
       it('retrieves the old value when not updated', async () => {
-        expect(await protocolFeeCache.getProtocolAumFeePercentageCache()).to.equal(AUM_PROTOCOL_FEE);
+        expect(await feeCache.getProtocolAumFeePercentageCache()).to.equal(AUM_PROTOCOL_FEE);
       });
 
       it('updates the cached value', async () => {
-        await protocolFeeCache.updateProtocolAumFeePercentageCache();
+        await feeCache.updateProtocolAumFeePercentageCache();
 
-        expect(await protocolFeeCache.getProtocolSwapFeePercentageCache()).to.equal(NEW_AUM_PROTOCOL_FEE);
+        expect(await feeCache.getProtocolAumFeePercentageCache()).to.equal(NEW_AUM_PROTOCOL_FEE);
```

Comment:
We were calling the wrong function here, and it just so happened that `NEW_AUM_PROTOCOL_FEE` was equal to `FIXED_PROTOCOL_FEE`. This is an example of why we should avoid reusing values.